build.sbt

```diff
@@ -9,6 +9,9 @@ ThisBuild / tlSonatypeUseLegacyHost := false
 ThisBuild / crossScalaVersions := Seq("3.1.3", "2.12.16", "2.13.8")
 
 ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("17"))
+ThisBuild / githubWorkflowOSes += "macos-latest"
+ThisBuild / githubWorkflowBuildMatrixExclusions +=
+  MatrixExclude(Map("os" -> "macos-latest", "project" -> "rootJVM"))
 
 val catsEffectVersion = "3.4-519e5ce-SNAPSHOT"
```

Comment:
Are `catsEffectVerson` and the line below it still up to date.
There have been a couple of `version` up/down(?) grades
recently.  Good reasons to get this PR in.

Reply:
You are right, they have fallen out of date. Fortunately CI runs on a merge commit so it will use the latest from `main`.